core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6230,6 +6230,7 @@ components:
             - $ref: '#/components/schemas/WorkspaceSettingSeparateSubmissionFinalOutputsConfig'
             - $ref: '#/components/schemas/WorkspaceSettingUseCromwellGcpBatchBackendConfig'
             - $ref: '#/components/schemas/WorkspaceSettingPubliclyReadableConfig'
+            - $ref: '#/components/schemas/WorkspaceSettingQuicksilverDataTablesConfig'
```

Comment:
Are we enshrining the name "quicksilver" into the code forever?  Will it make sense to future programmers?

Reply:
@calypsomatic I've renamed throughout to "compact data tables" - how does that sound?